setup-bastion-vars.env

```diff
@@ -1,3 +1,4 @@
 export BASTION_IP=`kubectl get services -o wide |grep bastion | awk '{print $4}'`
```

Comment:
can we put these env files in a folder?  In general, I want to make the root directory as short and clean as possible, so that people can see the README without scrolling over lots of files.

Reply:
y can do - FYI it's a half day to move and retest everything. Some cleanup work I want to do with the var names anyway.

Reply:
After you clean up the var names, I'm happy to move them for you (you can then review). Just let me know what tests you want to perform (or you can of course do the tests yourself).

Reply:
I've got it (the move) - there's a bunch more files to move around at the same time.

I'm thinking about two new subdirs: "scripts/" (for single-node and cluster configs) and "cluster-scripts/" (only cluster configs).